server/src/test/java/org/cloudfoundry/identity/uaa/provider/oauth/ExternalOAuthAuthenticationManagerIT.java

```diff
@@ -908,6 +909,37 @@ void updateShadowUser_IfAlreadyExists() {
         assertEquals(OriginKeys.UAA, uaaUser.getZoneId());
     }
 
+    @Test
+    void publishExternalGroupAuthorizationEvent_skippedIf_isRegisteredIdpAuthentication() {
```

Comment:
> publishExternalGroupAuthorizationEvent_skippedIf_isRegisteredIdpAuthentication

I thought this PR skips if it's not a "registered Idp Authentication"?

Reply:
Yes you're right. The method name is backwards. I'll fix it.